lib/bundles/inspec-compliance/cli.rb

```diff
@@ -145,11 +150,16 @@ def download(profile_name)
 
     desc 'upload PATH', 'uploads a local profile to Chef Compliance'
     option :overwrite, type: :boolean, default: false,
-      desc: 'Overwrite existing profile on Chef Compliance.'
+      desc: 'Overwrite existing profile on Server.'
+    option :owner, type: :string, required: false,
+      desc: 'Owner that should own the profile'
     def upload(path) # rubocop:disable Metrics/MethodLength, Metrics/AbcSize, PerceivedComplexity, Metrics/CyclomaticComplexity
       config = Compliance::Configuration.new
       return if !loggedin(config)
 
+      c = config.dup
```

Comment:
What was your thinking for needing to `dup` this here vs. just using the existing `config`? We don't return it to the caller, and everything else we pass `config` to appears to potentially need the `owner` and the rest of the `config` object anyways. It just seems unclear to me.

We should also be more explicit with our variable names. `c` isn't very descriptive but it's used throughout this method now.